.rat-excludes

```diff
@@ -74,5 +74,12 @@ logs
 .*scalastyle-output.xml
 .*dependency-reduced-pom.xml
 known_translations
+json_expectation
+local-1422981759269/*
```

Comment:
Are these timestamps here? If so, maybe this shouldn't be committed to the repo?